api-spec/STAC-query.yaml

```diff
@@ -0,0 +1,692 @@
+openapi: 3.0.1
```

Comment:
This is generated from the fragments, right? So no review required?

Reply:
correct